doc/taoMediaManagerApi.yml

```
@@ -38,6 +38,29 @@ paths:
           $ref: '#/components/responses/BadRequestResponse'
         500:
           $ref: '#/components/responses/InternalServerErrorResponse'
+  /taoMediaManager/SharedStimulus/update:
```

```suggestion
  /taoMediaManager/SharedStimulus/patch:
```

This should be a PATCH, now we want to change only the body, but we might want to update other resource fields.

---

i think we should not use the verbs to set up the uri, i believe update is more meaningful

---

I'm expecting `/taoMediaManager/SharedStimulus/patch` method to be accessible only with http patch method.